app/controllers/Api/v1/users_controller.rb

```diff
@@ -1,32 +1,10 @@
 class Api::V1::UsersController < ApplicationController
-  before_action :set_user, only: [:show, :update]
-  
-  def index
-    users = User.all
-    render json: users
-  end
-  
+  before_action :set_user, only: [:show]
+    
   def show
-    render json: @user
-  end
-
-  def create
-    @user = User.new(user_params)
-    if @user.save
-      render json: @user, status: :created
-    else
-      render json: @user.errors, status: :unprocessable_entity
-    end
+    render json: (@user ? @user : { error: "User not found" }), status: (@user ? :ok : :not_found)
```

Comment:
Nice! I would suggest splitting this up over multiple lines like this, makes it easier to parse. (I haven't tested it, so it's possible there's a syntax error here)

```suggestion
    render json: 
      @user ? @user : { error: "User not found" },
      status: @user ? :ok : :not_found
```

Reply:
Also, could we add spec to test this method?